controllers/app.controllers.js

```diff
@@ -0,0 +1,17 @@
+const { fetchTopics, fetchApi } = require("../models/app.models.js")
+const endPoints = require("../endpoints.json");
+
+exports.getTopics = (req, res, next) => {
```

Comment:
This is for the previous task and shouldn't be included in this PR.

Reply:
I have no idea how it got there...